common/lib/capa/capa/templates/choicegroup.html

```diff
@@ -1,43 +1,52 @@
-<form class="choicegroup capa_inputtype" id="inputtype_${id}">
+<form class="choicegroup capa_inputtype" id="${id}-form">
     <fieldset role="${input_type}group" aria-label="${label}">
+      % if response_data['label']:
```

Comment:
I wonder if there is a better fallback here?  When you use a fieldset, its not valid to omit the legend. I wonder if we could put a somewhat helpful default here instead, like ```else: <legend>Question</legend>``` It's not perfect, but at least we don't run the risk of omitting the ```<legend>``` all together?  Another useful side effect of this is that the word "Question" would be visible, and the author would likely NOT want it there, so they would make sure they use the appropriate markup.